crates/torii/indexer/src/processors/event_message.rs

```diff
@@ -28,6 +28,15 @@ where
         true
     }
 
+    fn task_priority(&self) -> usize {
+        1
```

Comment:
Isn't an enum more idiomatic, or you need logic around this priority, or a priority value seems more manageable?